Tests/Contour.RabbitMq.Tests/MessageHeaderSpecs.cs

```diff
@@ -443,6 +443,59 @@ public void ShouldFillOriginalMessageId()
                 Assert.IsTrue(message.Headers.ContainsKey(Headers.OriginalMessageId), "Должен быть установлен заголовок с индентификатором первого сообщения.");
             }
 
+            [Test]
+            public void ShouldPassCorellationIdOnEmit()
```

Comment:
It means that original correlation Id should be passed in headers without changing when emiting one-way message.